src/Aspire.Hosting.Azure.Sql/AzureSqlExtensions.cs

```diff
@@ -127,6 +128,18 @@ public static IResourceBuilder<AzureSqlDatabaseResource> AddDatabase(this IResou
         }
     }
 
+    /// <summary>
+    /// Configures the Azure SQL Database to be deployed with the specified SKU
```

Comment:
After a while you'll get used to these nit comments on PRs. Personally, I will never.

```suggestion
    /// Configures the Azure SQL Database to be deployed with the specified SKU.
```

Reply:
No worries, it is good to aim for perfection! :)